docs/configuring/chargebee-integration.md

```diff
@@ -0,0 +1,47 @@
+#Chargebee integration
+
+KuberLogic uses Webhooks to interact with the billing providers (such as Chargebee).
+
+###Configuring KuberLogic
+
+You should set Chargebee integration params in `kuberlogic/modules/dynamic-operator/config/manager/kustomization.yaml`
+
+```shell
+  - chargebee_site=
+  - chargebee_key=
+  - kuberlogic_domain=
```

Comment:
This parameter is not related to chargebee at all and it would make sense to note that it will be used as a domain for all services created via chargebee